watertap/property_models/NaCl_prop_pack.py

```diff
@@ -580,14 +582,16 @@ def _conc_mass_phase_comp(self):
             doc="Mass concentration",
         )
 
-        def rule_conc_mass_phase_comp(b, j):
+        def rule_conc_mass_phase_comp(b, p, j):
             return (
-                b.conc_mass_phase_comp["Liq", j]
-                == b.dens_mass_phase["Liq"] * b.mass_frac_phase_comp["Liq", j]
+                b.conc_mass_phase_comp[p, j]
+                == b.dens_mass_phase[p] * b.mass_frac_phase_comp[p, j]
```

Comment:
If the rule for conc_mass_phase_comp references the dens_mass_phase, should the index for dens_mass_phase remain ["Liq"] or should it be p?